src/python/pants/base/revision.py

```diff
@@ -24,7 +24,7 @@ def _parse_atom(cls, atom):
       return atom
 
   @classmethod
-  def semver(cls, rev):
+  def semver(cls, rev) -> "Revision":
```

Comment:
Is this a string because this is a `@classmethod`, so the type doesn't exist yet? I couldn't find any mention of string-types in Mypy's docs.

Reply:
Yes, exactly. In Python 3.7 you can use `from __future__ import annotations` to avoid having to use this unfortunate syntax.

Reply:
Thanks!